tests/molecules/He_param/CMakeLists.txt

```diff
@@ -25,6 +25,25 @@ if(NOT QMC_CUDA)
     SCALAR_VALUES
     HE_BSPLINE_PARAM)
 
+  # Check the parameter gradient code for the legacy driver
+  qmc_run_and_check_custom_scalar(
+    BASE_NAME
+    He_param_grad_legacy
+    BASE_DIR
+    "${qmcpack_SOURCE_DIR}/tests/molecules/He_param"
+    PREFIX
+    He_param_grad.param
+    INPUT_FILE
+    He_param_grad.xml
```

Comment:
Incorrect input file. This explains the coverage report complaints.
Can we rename the input file He_param_grad_legacy_driver.xml